binary-codec/README.md

```diff
@@ -0,0 +1,52 @@
+# Binary Codec
+
+This package contains functions to encode/decode to/from the [ripple binary serialization format](https://xrpl.org/serialization.html).
+
+## API
+
+### Encode
+
+```go
+encoded, err := binarycodec.Encode(jsonObject)
+```
+
+### Decode
+
+```go
+json, err := binarycodec.Decode(hexEncodedString)
+```
+### EncodeForMultisigning
+
+```go
+encoded, err := binarycodec.EncodeForMultisigning(jsonObject, xrpAccountID)
+```
+
+### EncodeForSigning
```

Comment:
What is the difference between `EncodeForSigning` and `Encode` ?

Reply:
The difference between these two is that `Encode` encodes every field of the transaction and `EncodeForSigning` removes the fields that are not required for signing